tmt/steps/provision/__init__.py

```diff
@@ -437,6 +438,20 @@ def _query_is_superuser(self, guest: 'Guest') -> Optional[bool]:
 
         return output.stdout.strip() == 'root'
 
+    def _query_is_ostree(self, guest: 'Guest') -> Optional[bool]:
+        # https://github.com/vrothberg/chkconfig/commit/538dc7edf0da387169d83599fe0774ea080b4a37#diff-562b9b19cb1cd12a7343ce5c739745ebc8f363a195276ca58e926f22927238a5R1334
+        output = self._execute(
+            guest,
+            Command(
```

Comment:
Why `Command` instead of Path.exists()?

Reply:
`Path.exists()` would tell you whether the path exists on the machine running tmt.

Reply:
Yeah, we need it for the guest.